probe_src/probe_frontend/python/ops.py

```diff
@@ -0,0 +1,361 @@
+
+# This file is automatically @generated by probe_macros
+
+import sys
+import typing
+from dataclasses import dataclass
+
+mod = sys.modules[__name__]
+
+@dataclass(init=False)
+class timespec:
```

Comment:
Can we "translate" the class-names into snakecase for Python? Apply a hard-coded constant Map[str, str], defaulting to the C name. I know the identifier comes from the C header, but when in Rome. I would also like to get rid of `tv_`, `ru_`, ... (hardcoded list of prefixes to remove is acceptable).

Reply:
> Can we "translate" the class-names into snakecase for Python?

sure, PascalCase to snake_case should be an easy heuristic.

> Apply a hard-coded constant Map[str, str], defaulting to the C name.

huh? I'm not sure what you mean by this, but I've been doing my best to avoid hard-coding any op names outside `lib/build.rs`

> I would also like to get rid of tv_, ru_, ... (hardcoded list of prefixes to remove is acceptable).

okay, also should be easy, my preferred way to implement this would be to strip any `/^[a-zA-Z0-9]+_/` from struct field names if all fields start with the same one.

Reply:
Whoops, by "snake case" I meant "Python's conventoins", which are Pascal case for class names, e.g. `class statx` should be `class Statx`.

Reply:
_**oh...**_   okay...   <sub>`git revert ...`</sub>

Reply:
Solved in https://github.com/charmoniumQ/PROBE/pull/20/commits/122952fc2f67aca953b5b394ca302eb9ee9d39c6 by making these changes to the generated rust structs, which then propagates to the python structs.